src/core/dfn.js

```diff
@@ -238,3 +243,32 @@ function addContractDefaults() {
     dfn.dataset.export = "";
   }
 }
+
+// - Sets data-defines on well-known definition content patterns
+function addDefinitionPointers() {
+  // A dl with class hasdefinitions marks all next siblings of dt with the class
+  // definition
+  /** @type NodeListOf<HTMLElement> */
+  const describedDTs = document.querySelectorAll("dl.hasdefinitions dt");
```

Comment:
Maybe `dl.definitions`?

Reply:
the reason I was liking "hasdefinitions" more than just "definitions" is that it is more easily distinguishable from the other pattern (it's easy to see that "hasdefinitions" is different from "definition", but it would really easy to confuse "definition" and "definitions" which generate quite different behavior).

Reply:
Makes sense, but still feels a little odd to me. Maybe I can get used to it :)

Reply:
Wasn't loving hasdefinitions either tbh. I updated the PR to use "definitions".